### PR TITLE
ASCBee

### PR DESCRIPTION
**Pull request by primebie1**
[FEATURE]
## About the PR 
Adds a new bee type: ASCBee, with custom sprites and text

## Why's this needed? 
Started as a method to learn DM and Goon's codebase, bloomed into a full bee, adds another tool to mapmaker's kit.

## Changelog
<!-- If necessary, put your changelog entry below. Otherwise, please delete it.
Use however you want to be credited in the changelog in place of CodeDude.
Use (*) for major changes and (+) for minor changes. For example: --!>
N/A

### CODE REVIEW

**Comment by ZeWaka** [comment]
Don't think this warrants a changelog since it's not implemented in the game at all.

**Comment by primebie1** [comment]
Yeah, makes sense, was mostly on autopilot for that XD

File: code/obj/critter/bee.dm [hunk 1]
```diff
@@ -712,6 +712,24 @@
 		icon_state = "sonicbee-wings"
 		sleeping_icon_state = "sonicbee-sleep"
 
+	ascbee
+		name = "ASCBee"
+		desc = "This bee looks rather... old school"
+		icon_body = "ascbee"
+		icon_state = "ascbee-wings"
+		sleeping_icon_state = "ascbee-sleep"
+		angertext = "beeps agressively at"
```
Comment (Gerhazo) [inline code/obj/critter/bee.dm:721]: typo review: probably should be "aggressively"

File: code/mob/living/critter/bee.dm [hunk 2]
```diff
@@ -992,4 +992,20 @@
 /mob/living/critter/small_animal/bee/beestation //A special bee that should allow non-admins to play as a bee.
 	non_admin_bee_allowed = 1
 
+/mob/living/critter/small_animal/bee/ascbee
+	name = "ASCBee"
+	desc = "This bee looks rather... old school"
```
Comment (Gerhazo) [inline code/mob/living/critter/bee.dm:997]: typo review: missing a period

File: code/obj/critter/bee.dm [hunk 3]
```diff
@@ -712,6 +712,24 @@
 		icon_state = "sonicbee-wings"
 		sleeping_icon_state = "sonicbee-sleep"
 
+	ascbee
+		name = "ASCBee"
+		desc = "This bee looks rather... old school"
```
Comment (Gerhazo) [inline code/obj/critter/bee.dm:717]: typo review: missing a period

**Pull request merged**